tests/pytorch/distributed/run_numerics.py

```diff
@@ -79,6 +79,14 @@ def main(argv=None, namespace=None):
     parser.add_argument("--quantization", type=str, default=None)
     args = parser.parse_args(argv, namespace)
 
+    # Quantization scheme
+    QUANTIZATION = args.quantization
```

Comment:
Silly bug, although I don't understand why this was working before:
```suggestion
    global QUANTIZATION
    QUANTIZATION = args.quantization
```

Reply:
Actually, we've already set `global QUANTIZATION` above. Don't think this does anything.